### ISSUE TITLE
A11y_SQL database projects extension_Advance Publish_Exclude Object Type_Name Role Value: Corresponding Name is not defined for checkboxes on the "Exclude object types" Page.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/azuredatastudio#22156]
### GitHub Tags
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #Name Role Value; #WCAG4.1.2; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #Screenreader;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### Environment Details
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps

1. Launch Azure Data Studio
2. Tab till extension and search for “SQL Database Projects” and install it.
3. Tab till database project and press enter.
4. “Database Project” panel will open.
5. Tab till “View And More Action” button and press enter.
6. Tab till “Create Project From Database” and press enter.
7. Tab till server field and press enter.
8. “Connection” dialog will open. Fill the details for SQL server and connect.
9. Tab till Create button and press enter.
10. Database project will get create and will appear in Database project panel.
11. Tab till created project name in Database Project panel.
12. Press context menu button(right click).
13. Tab till Publish and press enter.
14. Publish Project window will open.
15. Tab till “Advance” button and press enter.
16. Tab till Exclude Object Type tab and press enter.
17. Verify whether the Corresponding Name is defined for checkboxes on the "Exclude object types" Page or not.

### Actual Result
Corresponding Name is not defined for checkboxes also NVDA not reading the checkbox status whether they are checked or unchecked.

### Expected Result
Corresponding Name should be defined for checkboxes and NVDA should announce the status of checkboxes.

### User Impact
Screen reader user will not be able to get the proper information and status of the checkboxes.

Attachment

![Presentation3](https://user-images.githubusercontent.com/124244478/223043385-3b2f05bc-da2c-4621-ad01-4666f1138e19.gif)

**Comment by kisantia** [comment]
@AnKushSingh05 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!
